aws/efs/deploy/rbac.yaml

```diff
@@ -34,6 +34,7 @@ kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: leader-locking-efs-provisioner
+  namespace: default
```

Comment:
put the same comment here?

# replace with namespace where provisioner is deployed